cartography/data/jobs/analysis/azure_storage_asset_exposure.json

```diff
@@ -15,6 +15,11 @@
 			"iterative": true,
 			"iterationsize": 100
 		},
+		{
+			"query": "MATCH ((n)<-[:CONTAINS]-(:AzureStorageQueueService|AzureStorageTableService|AzureStorageFileService)<-[:USES]-(:AzureStorageAccount)<-[:RESOURCE]-(:AzureSubscription{id: $AZURE_SUBSCRIPTION_ID})<-[:RESOURCE]-(:AzureTenant{id: $AZURE_TENANT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID})) WHERE EXISTS(n.exposed_internet) AND n:AzureStorageQueue OR n:AzureStorageTable OR n:AzureStorageFileShare WITH n LIMIT $LIMIT_SIZE REMOVE n.exposed_internet, n.exposed_internet_type RETURN COUNT(*) as TotalCompleted",
```

Comment:
`(:AzureStorageQueueService|AzureStorageTableService|AzureStorageFileService)` this or statement support in 5.* version and we are using 4.* version. Implement for 4.*